jetty9-base/pom.xml

```diff
@@ -112,7 +136,7 @@
               <goal>copy-dependencies</goal>
             </goals>
             <configuration>
-              <outputDirectory>${project.build.directory}/jetty-base/lib/gcp</outputDirectory>
+              <outputDirectory>${project.build.directory}/jetty-base/lib/gcloud</outputDirectory>
```

Comment:
AFAIK this came in with the changes I merged from master, it is not something I have introduced.